api.js

```diff
@@ -132,11 +132,8 @@ Api.prototype.observationUpdate = function (req, res, m) {
         })
         opts.links = old.id
       }
-      var newObs = Object.assign(old, {
-        properties: obs.properties,
-        lat: obs.lat,
-        lon: obs.lon
-      })
+      var newObs = Object.assign(old, {})
```

Comment:
I'm confused. Is old the observation that is in the database? How does it get updated with what gets passed in? Seems like you're assigning newObs to be the old db observation with the only update being the removal of the version? Unless I'm reading this wrong...

Reply:
ya you're right I should have had more coffee before doing this

Reply:
Woah! Very troubling if the tests were passing!